### ISSUE TITLE
Unable to parse mysql ddl with table configuration

### ISSUE THREAD

**Issue by vegetablest** [xnuinside/simple-ddl-parser#206]
**Describe the bug**
If the MySQL table creation statement specifies table configuration such as storage engine or character encoding, the content will not be parsed.

**To Reproduce**
1.Parsed failed
```sql
CREATE TABLE `employee` (
  `user_id` int(11) NOT NULL AUTO_INCREMENT,
  `user_name` varchar(50) NOT NULL,
  `authority` int(11) DEFAULT '1' COMMENT 'user auth',
  PRIMARY KEY (`user_id`),
  KEY `FK_authority` (`user_id`,`user_name`)
) ENGINE=InnoDB AUTO_INCREMENT=10 DEFAULT CHARSET=utf8;
```
2.Parsed successfully
```sql
CREATE TABLE `employee` (
  `user_id` int(11) NOT NULL AUTO_INCREMENT,
  `user_name` varchar(50) NOT NULL,
  `authority` int(11) DEFAULT '1' COMMENT 'user auth',
  PRIMARY KEY (`user_id`),
  KEY `FK_authority` (`user_id`,`user_name`)
);
```
![image](https://github.com/xnuinside/simple-ddl-parser/assets/53895794/4a85f0d9-5264-495d-9f1e-d5b28f4f3c3c)

**Expected behavior**
I think both parsing should be able to parse the field information.

**Additional context**
python==3.11     simple-ddl-parser==0.31.0

**Comment by utkarshkoppikar7** [comment]
I have noticed the same behaviour, should we have procedure to filter these properties out?

**Comment by xnuinside** [comment]
Fix was released in version 1.2.0 - https://pypi.org/project/simple-ddl-parser/ https://github.com/xnuinside/simple-ddl-parser/pull/247. @vegetablest thanks for reporting this issue, @utkarshkoppikar7 thanks for comment, it was good push to work on it ) . test was added: https://github.com/xnuinside/simple-ddl-parser/blob/main/tests/dialects/test_mysql.py#L550. thanks one more time! if will be any other issues - feel free to open new one

**Comment by vegetablest** [comment]
awesome👍
